src/module/apps/compendium-browser/tabs/spell.ts

```diff
@@ -41,11 +29,11 @@ export class CompendiumBrowserSpellTab extends CompendiumBrowserTab {
         const indexFields = [
             "img",
             "system.level.value",
-            "system.category.value",
-            "system.traditions.value",
```

Comment:
Given that category filters are completely gone, I think the only thing you need to try to make backwards compatible here is traditions. It should be doable, but we do have compendium migrations now so I don't think its necessary anymore. You can resolve this if that's your angle.

We should probably start saving min versions for each tab so that the compendium browser shows warnings in the sources tab at some point.